apps/DesktopStreamer/main.cpp

```diff
@@ -40,14 +40,16 @@
 
 #include "MainWindow.h"
 
+#include <deflect/utils.h>
 #include <deflect/version.h>
 
 #include <QCommandLineParser>
 
 int main(int argc, char* argv[])
 {
-    QApplication app(argc, argv);
+    deflect::blockFalseWarnings();
```

Comment:
Maybe... I wasn't too sure about this. If we place it somewhere in Stream (e.g. constructor) it will overwrite a previously set qInstallMessageHandler() and there is no getter to check beforehand. But it seems that the setter returns the previous handler so we could try to set the new one and if the returned value is not nullptr set back the previous one. A bit fishy but I can try.